contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,192 @@
+     column of order_table_2 that are not present in the CUSTOMER_ID column of customer_table.
+
+     "order_table_2": {
+         "ORDER_ID": ["ddd", "eee", "fff"],
+         "CUSTOMER_ID": [1, 4, 5],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+     ExpectColumnValuesToBePresentInAnotherTable will PASS for example 1 and FAIL for example 2.
+
+     Args:
+         template_dict: dict containing the following keys:
+             foreign_key_column: foreign key-column of current table that we want to validate.
+             foreign_table: foreign table.
+             primary_key_column_in_foreign_table: key column for primary key in foreign table.
+    """
+
+    library_metadata = {
+        "maturity": "experimental",
+        "tags": ["table expectation", "multi-table expectation", "query-based"],
+        "contributors": [
+            "@great_expectations",
+        ],
+        "requirements": [
+            "sqlalchemy",
+            "snowflake-sqlalchemy",
+            "snowflake-connector-python",
+        ],
+        "has_full_test_suite": False,
+        "manually_reviewed_code": True,
+    }
+
+    metric_dependencies = ("query.template_values",)
+    template_dict: dict
+    query = """
+        SELECT count(1) FROM (
+        SELECT a.{foreign_key_column}
+        FROM {active_batch} a
+        LEFT JOIN {foreign_table} b
+            ON a.{foreign_key_column} = b.{primary_key_column_in_foreign_table}
+        WHERE b.{primary_key_column_in_foreign_table} is NULL)
+        """
+    success_keys = (
+        "template_dict",
+        "query",
+    )
+    domain_keys = ("query", "batch_id", "row_condition", "condition_parser")
+
+    default_kwarg_values = {
+        "catch_exceptions": False,
+        "meta": None,
+        "query": query,
+    }
+
+    def _validate_template_dict(
+        self, configuration: Optional[ExpectationConfiguration] = None
+    ) -> None:
+        template_dict = configuration.kwargs.get("template_dict")
+        if not isinstance(template_dict, dict):
+            raise TypeError("template_dict must be supplied as a dict")
+        if not all(
+            [
+                "foreign_key_column" in template_dict,
+                "foreign_table" in template_dict,
+                "primary_key_column_in_foreign_table" in template_dict,
+            ]
+        ):
+            raise KeyError(
+                "The following keys have to be in the template dict: id, foreign_table, foreign_key_column"
+            )
+
+    def _validate(
+        self,
+        metrics: dict,
+        runtime_configuration: Optional[dict] = None,
+        execution_engine: Optional[ExecutionEngine] = None,
+    ) -> Union[ExpectationValidationResult, dict]:
+        configuration = self.configuration
+        self._validate_template_dict(configuration)
+        as_dict_as_template = metrics.get("query.template_values")[0]
+        final_value = list(as_dict_as_template.values())[0]
```

Comment:
Do we know the key to pull out of `as_dict_as_template`? Might be clearer to reference via that key rather than convert to a list and use the index.

Reply:
Thanks for this. The line has been updated to 

```python
final_value = metrics.get("query.template_values")[0]["COUNT(1)"]
```